pkg/support/collectors.go

```diff
@@ -102,3 +113,43 @@ func (c *collectorFactory) DefaultCollectors() []*Collect {
 		},
 	}
 }
+
+func (c *collectorFactory) EksaHostCollectors(osFamilies map[v1alpha1.OSFamily]bool) []*Collect {
+	for family := range osFamilies {
+		switch family {
+		case v1alpha1.Ubuntu:
+			return c.ubuntuHostCollectors()
+		}
+	}
+	return nil
+}
+
+func (c *collectorFactory) ubuntuHostCollectors() []*Collect {
+	return []*Collect{
```

Comment:
Would it make sense to use an array or map here that could be easier to add additional filenames/paths to instead of duplicating the analyzer object over and over?

Reply:
yeah , we could simplify this. for the deployments, [we're using a simplified struct that has just the variable information in it](https://github.com/aws/eks-anywhere/blob/main/pkg/support/analyzers.go#L19), or we could use a map that's [collector name]collectionPath, and just set everything else the same. The only field that's really variable at this point is timeout, and the smaller collectors won't be harmed by having a larger timeout

Reply:
we can also easily update this after this PR merges